lib/workers/repository/update/pr/body/index.ts

```diff
@@ -1,6 +1,7 @@
 import type { RenovateConfig } from '../../../../../config/types';
 import type { PrDebugData } from '../../../../../modules/platform';
 import { platform } from '../../../../../modules/platform';
+import { BITBUCKET_WEB_ENDPOINT } from '../../../../../modules/platform/bitbucket';
```

Comment:
I dont _think_ that will work since its not the renovate platform, but the specific update platform/host that I need to check

eg: renovate running on a GitHub repository with an update for a Bitbucket-hosted package

Reply:
In theory it could be independent of platform. In practice right now the overlap is 100%.

Here's a question: the way things are today, would we ever identify `sourceDirectory` for a host which _isn't_ a Renovate `platform` module? I think that we subsequently rely on platform functions once sourceRepo and sourceDirectory are identified, don't we? i.e. we don't have the concept of platform-independent changelogs

Reply:
Agreed